### DISCUSSION TITLE
Quasar site is down?

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14150] [General - Components / Directives / etc] [answered]
Dear,

I can't access https://quasar.dev, is it down?

**Comment by pdanpdan** [comment]
https://www.google.com/search?q=is+site+down&pws=0&gl=us&gws_rd=cr

**Reply by 549393092** [reply]
@pdanpdan sorry, I don't get your point. Am I using some words not properly?

**Comment by pdanpdan** [comment]
There are lots of sites that offer that service for you

**Comment by smolinari** [comment]
It's working for me. 

Scott

**Comment by smolinari** [comment, marked as answer]
But, what @pdanpdan is getting at is there are a number of sites you can use to answer your question, instead of asking it here. 

like this: https://www.isitdownrightnow.com/quasar.dev.html

Scott

**Comment by 549393092** [comment]
@smolinari I got it and thanks for your help!
